src/pages/SignIn/SignIn.jsx

```diff
@@ -1,14 +1,19 @@
 import UserSignIn from './components/UserSignIn';
 import GoogleSignInButton from './components/GoogleSignInButton';
 import UserSignUp from '../../components/UserSignUp';
+import Text from '../../components/Text';
 
 const SignIn = () => {
   return (
     <section className="Login">
-      <h1>Log In</h1>
-      <GoogleSignInButton />
-      <UserSignIn />
-      <UserSignUp />
+      <Text h1 className="font-semibold p-5">
+        Log In
+      </Text>
+      <div className="flex w-[400px] mx-auto flex-col flex-wrap mb-6 md:mb-0 mt-6 md:mt-2 gap-4">
```

Comment:
@rainawan will you able to change w-[400px] to max-w-[400px] on your next pr pls? :)

Reply:
<img width="302" alt="image" src="https://github.com/rainawan/BruinDictionary/assets/96547645/1f19a110-3e03-413d-b767-d65e2eda2e8c">